src/docs/asciidoc/deployment/configuration/configuration.adoc

```diff
@@ -39,6 +39,7 @@ See link:{mongo_doc}connection-string/[mongo connection string] for the complete
 
 This configuration is mandatory (no default configuration is provided).
 
+Operator Fabric is supported mongo version 5.0.0 and above.
```

Comment:
A little error, here is a suggestion :
Operator Fabric is supporting mongo versions 5.0.0 and above.